### PR TITLE
Add a function for /simple/supported_vs_currencies coingecko endpoint #19

### PR DESCRIPTION
**Pull request by WHYTEWYLL**

### CODE REVIEW

File: packages/coingecko/recipes/e2e.json [hunk 1]
```diff
@@ -4,6 +4,8 @@
   },
   {
     "query": "./get.graphql",
```
Comment (Niraj-Kamdar) [inline packages/coingecko/recipes/e2e.json:6]: This is just what I put together quickly while bootstrapping but we would like to have separate files for each queries.

File: packages/coingecko/src/query/routes/ping.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  }
+
+  const json = <JSON.Obj>JSON.parse(response.body);
+  const geckoSays = json.getString("gecko_says");
+  if (geckoSays) {
+    return {
+      gecko_says: geckoSays.valueOf(),
+    };
+  }
+  throw Error(response.statusText);
```
Comment (Niraj-Kamdar) [inline packages/coingecko/src/query/routes/ping.ts:27]:
```suggest
  throw Error("Invalid response body!");
```

File: packages/coingecko/src/query/routes/supportedVSCurrencies.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      },
+    });
+    if (!response || response.status !== 200 || !response.body) {
+      throw Error(response.statusText);
+    }
+  
+    const jsonArray = <JSON.Arr>JSON.parse(response.body);
+  
+    if (!jsonArray) {
+      throw Error(response.statusText);
```
Comment (Niraj-Kamdar) [inline packages/coingecko/src/query/routes/supportedVSCurrencies.ts:24]:
```suggest
      throw Error("Invalid response body!");
```

**Review by Niraj-Kamdar — CHANGES_REQUESTED** [review]

File: packages/coingecko/src/query/routes/supportedVSCurrencies.ts (hunk 3 above)
Comment (WHYTEWYLL) [reply packages/coingecko/src/query/routes/supportedVSCurrencies.ts:24]: Changes done, for #60

File: packages/coingecko/recipes/endpoints/supportCurrencies/e2e.json [hunk 4]
```diff
@@ -0,0 +1,11 @@
+[
+  {
+    "api": "ens/testnet/coingecko.eth"
+  },
+  {
+    "query": "./getSupportCurrencies.graphql",
```
Comment (Niraj-Kamdar) [inline packages/coingecko/recipes/endpoints/supportCurrencies/e2e.json:6]: It has to be same as the name of schema file: `packages/coingecko/recipes/endpoints/supportCurrencies/schema.graphql`

**Review by Niraj-Kamdar — CHANGES_REQUESTED** [review]

File: packages/coingecko/recipes/endpoints/supportCurrencies/e2e.json (hunk 4 above)
Comment (WHYTEWYLL) [reply packages/coingecko/recipes/endpoints/supportCurrencies/e2e.json:6]: Done!

**Pull request merged**